tests/e2e/e2e_exec_test.go

```diff
@@ -617,7 +617,7 @@ func (s *IntegrationTestSuite) executeGaiaTxCommand(ctx context.Context, c *chai
 	}
 }
 
-func (s *IntegrationTestSuite) executeHermesCommand(ctx context.Context, hermesCmd []string) (string, string) {
+func (s *IntegrationTestSuite) executeHermesCommand(ctx context.Context, hermesCmd []string) ([]byte, []byte) {
```

Comment:
Why change from string to bytes here? You have a string comparison later on.

Reply:
this is for unmarshal json from byte, otherwise, unmarshal has to convert string to byte